### DISCUSSION TITLE
add fullcalendar plugin

### DISCUSSION THREAD

**Discussion by ahmed-abess** [directus/directus#13644] [Archive]
**when i try to use fullcalendar in module vue iget not found page**

**Comment by ahmed-abess** [comment]
just when write
import '@fullcalendar/core/vdom'
import FullCalendar from "@fullcalendar/vue"
import dayGridPlugin from '@fullcalendar/daygrid'
import interactionPlugin from '@fullcalendar/interaction'

**Comment by rijkvanzanten** [comment]
Please don't open duplicate discussions (https://github.com/directus/directus/discussions/13636)
